products/statement-generator/src/components/FormHeader.tsx

```diff
@@ -167,7 +168,7 @@ const FormHeader = () => {
   return (
     <div className={classes.outerWrapper}>
       <div className={classes.formHeader}>
-        <h3>{formTitle}</h3>
+        <h3 className="formTitle">{formTitle}</h3>
```

Comment:
rather than importing an external css file and creating a 'formTitle' class inline, can you use Material UI and create a class that way?